src/AssemblyAI/Core/JsonConfiguration.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using AssemblyAI.Lemur;
+using OneOf;
+
+namespace AssemblyAI.Core;
+
+public static class JsonOptions
+{
+    public static readonly JsonSerializerOptions JsonSerializerOptions;
+
+    static JsonOptions()
+    {
+        JsonSerializerOptions = new JsonSerializerOptions
+        {
+            Converters =
+            {
+                new DateTimeSerializer(),
+                new OneOfSerializer<OneOf<LemurStringResponse, LemurQuestionAnswerResponse>>()
+            },
+            WriteIndented = true,
+            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
```

Comment:
I had to configure this, so that `null` properties are omitted from the JSON.
Otherwise there's an error in the API.

The API should be able to handle null values, but it doesn't properly for some properties, which has been reported.